recipes/trf/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: trf
+  version: "4.07b"
+
+source:
+  fn: trf407b
+  url: https://tandem.bu.edu/trf/downloads/trf407b.linux64 # [linux]
```

Comment:
Btw. there is a [linux64] trick, just not sure how it is called. But linux is also linux-32